module/event-sourcing/src/Infrastructure/Projector/ProjectorProvider.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types = 1);
+
+namespace Ergonode\EventSourcing\Infrastructure\Projector;
+
+use Ergonode\SharedKernel\Domain\DomainEventInterface;
+
+class ProjectorProvider
```

Comment:
Do you think it makes sense to have a separate "container" for those projectors? The processor could just receive them

Reply:
In my opinion, the processor and the provider have a different responsibility.
The provider delivers the respective projects according to the given conditions. 
The processor's is to operate them